src/faq/troubleshooting.rst

```diff
@@ -0,0 +1,222 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _faq/troubleshooting:
+
+===============
+Troubleshooting
+===============
+
+Where are the CouchDB log files located?
+----------------------------------------
+
+For a default Linux/Unix installation the log files are located here::
+
+    /usr/local/var/log/couchdb/couch.log
+
+This is set in the default.ini file located here::
+
+    /etc/couchdb/default.ini
+
+If you've installed from source and are running CouchDB in dev mode the log
+files are located here::
+
+    YOUR-COUCHDB-SOURCE-DIRECTORY/tmp/log/couch.log
```

Comment:
Also no longer true.

If you've installed from source and are running CouchDB with the developer `dev/run` utility, log files are under the `dev/logs` directory.